src/Toolkit/Toolkit.WinUI/UI/Controls/FloorFilter/FloorFilter.Theme.xaml

```diff
@@ -4,6 +4,9 @@
     xmlns:controls="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
     xmlns:internal="using:Esri.ArcGISRuntime.Toolkit.Internal"
     xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="../../../Themes/SharedResources.xaml" />
+    </ResourceDictionary.MergedDictionaries>
```

Comment:
Right approach, but this should be in the first part of Generic.xaml (before FloorFilter.Theme.xaml or anything else relying on the resources is referenced). The ensures resources aren't imported twice by any xaml files.

Reply:
I tried that approach a few different ways, but I see crashes for missing keys at run time.

Reply:
In that case these resources aren't shared, and should be added to `FloorFilter.Theme.xaml`.